Source/RapyutaSimulationPlugins/Public/Core/RRROS2GameMode.h

```diff
@@ -58,20 +59,27 @@ class RAPYUTASIMULATIONPLUGINS_API ARRROS2GameMode : public AGameMode
     TSubclassOf<URRROS2SimulationStateClient> ROS2SimStateClientClass = URRROS2SimulationStateClient::StaticClass();
 
     /**
-     * @brief Set timestep
+     * @brief Set timestep by FApp::SetFixedDeltaTime.
      *
      * @param InFrequency game time update frequency, timestep become 1/frequency.
+     * @attention FApp::SetFixedDeltaTime take arg as double but InFreqency is float to be able to access from BP.
      */
     UFUNCTION(BlueprintCallable)
-    virtual void SetFixedTimestep(float InStepSize);
+    virtual void SetFixedTimeStep(const float InStepSize);
```

Comment:
out of curiosity : is there an advantage in using `const` for a float input argument of a function?

Reply:
- clarify the usage of arguments/function
- optimization hint for compiler

I think const is not required for this func but kind of my habit to use const as much as possible.